example/lib/src/screen/home_page.dart

```diff
@@ -138,9 +156,11 @@ class _HomePageState extends State<HomePage>
 
   void _signIn() async {
     try {
+      final requestCode = int.tryParse(_codeEditingController.text);
+      final loginOption = LoginOption(_isOnlyWebLogin, "normal", requestCode: requestCode);
```

Comment:
put a fixed value here for requestCode, and add comments saying that this is not a must.
(just like what we did with "normal" for botprompt value).